lib/containers/utils.pm

```diff
@@ -190,11 +190,10 @@ sub test_built_img {
 
 C<can_build_sle_base> should be used to identify if sle base image runs against a
 system that it does not support registration and SUSEConnect.
-In this case the build of the container engine is not going to work as the base images
+In this case the build of the base image is not going to work as it lacks the repositories
 lacks of the repos.
```

Comment:
you can remove this line as it's mentioned above

Reply:
done